packages/strapi-plugin-documentation/services/Documentation.js

```diff
@@ -619,6 +619,18 @@ module.exports = {
             minItems: min,
             maxItems: max,
           };
+        } else if (['datetime', 'date', 'password'].includes(type)) {
+          acc.properties[current] = {
+            type: 'string',
+            format: this.getFormat(type),
+            description,
+            default: defaultValue,
+            minimum,
+            maxmimun,
+            maxLength,
+            minLength,
+            enum: enumeration,
+          };
```

Comment:
I would suggest to just modify the `else` block.
Something like:
```
        } else {
          acc.properties[current] = {
            type,
            format: getFormat(attribute.type),
            description,
            default: defaultValue,
            minimum,
            maxmimun,
            maxLength,
            minLength,
            enum: enumeration,
          };
        }
```

(In your code the value of `type` is `'string'` for date and password, so it would not work for those two. The right variable to use would be `attribute.type`.)

(I think we need to modify the function `getType` so it handles `datetime`)

Reply:
Updated. Thanks for the comment, the code looks much better now.